creator-node/src/routes/nodeSync.js

```diff
@@ -16,14 +16,19 @@ module.exports = function (app) {
   app.get('/export', handleResponse(async (req, res) => {
     const walletPublicKeys = req.query.wallet_public_key // array
 
+    const t = await models.sequelize.transaction()
+
     // Fetch cnodeUser for each walletPublicKey.
-    const cnodeUsers = await models.CNodeUser.findAll({ where: { walletPublicKey: walletPublicKeys } })
+    const cnodeUsers = await models.CNodeUser.findAll({ where: { walletPublicKey: walletPublicKeys }, transaction: t })
     const cnodeUserUUIDs = cnodeUsers.map((cnodeUser) => cnodeUser.cnodeUserUUID)
 
     // Fetch all data for cnodeUserUUIDs: audiusUsers, tracks, files.
-    const audiusUsers = await models.AudiusUser.findAll({ where: { cnodeUserUUID: cnodeUserUUIDs } })
-    const tracks = await models.Track.findAll({ where: { cnodeUserUUID: cnodeUserUUIDs } })
-    const files = await models.File.findAll({ where: { cnodeUserUUID: cnodeUserUUIDs } })
+    const [audiusUsers, tracks, files] = await Promise.all([
+      models.AudiusUser.findAll({ where: { cnodeUserUUID: cnodeUserUUIDs }, transaction: t }),
+      models.Track.findAll({ where: { cnodeUserUUID: cnodeUserUUIDs }, transaction: t }),
+      models.File.findAll({ where: { cnodeUserUUID: cnodeUserUUIDs }, transaction: t })
```

Comment:
This is an implicit `where <value> in <array>` clause. I would use the [sequelize operators](https://sequelize.org/master/manual/querying.html#operators) to specify explicitly that it's an or